src/SFA.DAS.SharedOuterApi/Infrastructure/HealthCheck/ApprenticeAccountsApiHealthCheck.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Api.Common.Infrastructure;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.Infrastructure.HealthCheck
+{
+    public class ApprenticeAccountsApiHealthCheck : IHealthCheck
+    {
+        private const string HealthCheckResultDescription = "Apprentice Accounts Api check";
+
+        private readonly IApprenticeCommitmentsApiClient<ApprenticeAccountsApiConfiguration> _apiClient;
+        private readonly ILogger<ApprenticeAccountsApiHealthCheck> _logger;
+
+        public ApprenticeAccountsApiHealthCheck(IApprenticeCommitmentsApiClient<ApprenticeAccountsApiConfiguration> apiClient, ILogger<ApprenticeAccountsApiHealthCheck> logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("Pinging ApprenticeAccounts API");
+
+            var timer = Stopwatch.StartNew();
+            var response = await _apiClient.GetResponseCode(new GetPingRequest());
+            timer.Stop();
+
+            if ((int)response == 200)
+            {
+                var durationString = timer.Elapsed.ToHumanReadableString();
+
+                _logger.LogInformation($"ApprenticeAccounts API ping successful and took {durationString}");
+
+                return HealthCheckResult.Healthy(HealthCheckResultDescription,
+                    new Dictionary<string, object> { { "Duration", durationString } });
+            }
+
+            _logger.LogWarning($"ApprenticeAccounts API ping failed : [Code: {response}]");
+            return HealthCheckResult.Unhealthy(HealthCheckResultDescription);
+        }
+    }
+}
```

Comment:
I'm a little confused about this endpoint, this healthcheck also exists in the apprenticeportal infrastructure site, so I suspect this one isn't being used (probably why the issue above isn't causing problems). 

Just to add to the confusion I have also added this heathcheck here in the approvals APIM rework. But we can take whichever arrives first in master

Reply:
yes the one in the portal is there as I was copying the commitments setup - ive removed it from the portal as its not used